src/schema/nmdc.yaml

```
@@ -955,9 +955,25 @@ enums:
         aliases:
           - bottom_layer
       organic_layer:
+        title: Organic layer
+        description: The portion of a mixture containing dissolved organic material
       aqueous_layer:
+        title: Aqueous layer
+        description: The portion of a mixture containing molecules dissolved in water
+        aliases:
+          - water layer
```

```suggestion
          - water_layer
```

I'm not sure exactly how aliases work in linkml - do we need both PVs?